aws-wafv2-loggingconfiguration/aws-wafv2-loggingconfiguration.json

```diff
@@ -0,0 +1,258 @@
+                  "type": "object"
+                },
+                "IncludedPaths": {
+                  "description": "Match only the specified include paths. See also MatchScope in JsonBody.",
+                  "type": "array",
+                  "minItems": 1,
+                  "items": {
+                    "type": "string"
+                  }
+                }
+              }
+            },
+            "MatchScope": {
+              "description": "The parts of the JSON to match against using the MatchPattern. If you specify All, AWS WAF matches against keys and values. ",
+              "type": "string",
+              "enum": [
+                "ALL",
+                "KEY",
+                "VALUE"
+              ]
+            }
+          },
+          "required": [
+            "MatchPattern",
+            "MatchScope"
+          ]
+        },
+        "Method": {
+          "description": "Inspect the HTTP method. The method indicates the type of operation that the request is asking the origin to perform. ",
+          "type": "object"
+        },
+        "QueryString": {
+          "type": "object",
+          "description": "Inspect the query string. This is the part of a URL that appears after a ? character, if any. "
+        },
+        "SingleHeader": {
+          "description": "Inspect a single header. Provide the name of the header to inspect, for example, User-Agent or Referer. This setting isn't case sensitive.",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "Name": {
+              "description": "The name of the query header to inspect.",
+              "type": "string"
+            }
+          },
+          "required": [
+            "Name"
+          ]
+        },
+        "UriPath": {
+          "type": "object",
+          "description": "Inspect the request URI path. This is the part of a web request that identifies a resource, for example, /images/daily-ad.jpg. "
+        }
+      }
+    }
+  },
+  "properties": {
+    "ResourceArn": {
+      "description": "The Amazon Resource Name (ARN) of the web ACL that you want to associate with LogDestinationConfigs.",
+      "type": "string"
+    },
+    "LogDestinationConfigs": {
+      "description": "The Amazon Kinesis Data Firehose Amazon Resource Name (ARNs) that you want to associate with the web ACL.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "RedactedFields": {
+      "description": "The parts of the request that you want to keep out of the logs. For example, if you redact the HEADER field, the HEADER field in the firehose will be xxx.",
+      "type": "array",
+      "insertionOrder": false,
+      "items": {
+        "$ref": "#/definitions/FieldToMatch"
+      }
+    },
+    "ManagedByFirewallManager": {
```

Comment:
We should set this field as a `readOnlyProperties`.

Reply:
done